src/components/checkbox/__control/checkbox__control.bemhtml

```diff
@@ -0,0 +1,16 @@
+block('checkbox').elem('control')(
+    tag()('input'),
+
+    attrs()(function() {
+        const originalAttrs = applyNext();
+
+        originalAttrs.autoComplete = originalAttrs.autocomplete;
```

Comment:
can't we pass autoComplete as an attribute? what's the point of this template btw?